reconcile/utils/ocm.py

```diff
@@ -67,6 +68,7 @@ def __init__(
         self.access_token_client_id = access_token_client_id
         self.access_token_url = access_token_url
         self.offline_token = offline_token
+        self.session = requests.Session()
```

Comment:
nit: maybe make this `self._session`? I know we don't have the strongest convention of public vs. private in the code base, but not a bad place to start.

Reply:
done